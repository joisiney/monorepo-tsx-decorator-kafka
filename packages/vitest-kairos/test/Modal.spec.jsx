import { render } from '@testing-library/react-native'
import { Modal, Text } from 'react-native'
import { test } from 'vitest'

const View = () => {
  return (
    <Modal>
      <Text>Hello, world!</Text>
    </Modal>
  )
}

test('scroll view components render correctly', () => {
  const container = render(<View />)
  expect(container).toMatchSnapshot()
})