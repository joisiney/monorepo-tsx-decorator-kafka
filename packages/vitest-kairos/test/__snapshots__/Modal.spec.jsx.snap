// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`scroll view components render correctly 1`] = `
<Modal
  hardwareAccelerated={false}
  visible={true}
>
  <Text>
    Hello, world!
  </Text>
</Modal>
`;
